pwndbginit/pwndbg_lldb.py

```diff
@@ -121,15 +94,62 @@ def main():
         print("[-] Launcher: Initializing Pwndbg")
     lldbinit.main(debugger, lldb_version[0], lldb_version[1], debug=debug)
 
-    from pwndbg.dbg.lldb.repl import PwndbgController
-    from pwndbg.dbg.lldb.repl import print_error
-    from pwndbg.dbg.lldb.repl import print_warn
     from pwndbg.dbg.lldb.repl import run as run_repl
 
     if debug:
         print("[-] Launcher: Entering Pwndbg CLI")
 
-    # Prepare the startup commands.
+    run_repl(controller, *args, debug=debug)
+
+    # Dispose of our debugger and terminate LLDB.
+    lldb.SBDebugger.Destroy(debugger)
+    lldb.SBDebugger.Terminate()
+
+
+def get_venv_bin_path():
+    bin_dir = "Scripts" if os.name == "nt" else "bin"
```

Comment:
What's that btw? Is NT some kind of support for Windows? :)

Reply:
I'm not sure. I think that was introduced in #3119.